MAINTAINERS.md

```diff
@@ -0,0 +1,14 @@
+Document that refers to a specific project within the Parca ecosystem
+is maintained by the maintainers of the respective project. For example, refer
+to the maintainers specified in Parca Agent's
+[MAINTAINERS.md](https://github.com/parca/parca-agent/blob/main/MAINTAINERS.md)
+file for the Parca Agent.
+
+Note that the documentation for the Parca server is located in the [parca.dev website](https://www.parca.dev/docs/overview).
+
+Refer to the following maintainers with their focus areas:
+
+* Frederic Branczyk <https://github.com/brancz> @brancz: Everything related to the Parca (storage, backend, frontend).
```

Comment:
Everything related to the Parca server?

or

Everything related to the Parca project?